gcloud-java-resourcemanager/src/main/java/com/google/gcloud/resourcemanager/ResourceManager.java

```diff
@@ -27,5 +32,79 @@
 
   public static final String DEFAULT_CONTENT_TYPE = "application/octet-stream";
 
-  // TODO(ajaykannan): Fix me! Add in missing methods.
+  /**
+   * Create a new project.
```

Comment:
we should either copy or reference the description from the methods.
e.g. https://cloud.google.com/resource-manager/reference/rest/v1beta1/projects/create
https://cloud.google.com/resource-manager/reference/rest/v1beta1/projects/delete, ,